conventions/documentation/badges.md

```diff
@@ -30,7 +59,8 @@ A status badge is a PatternFly badge that includes both a status icon and status
 
 ## Tech preview badge
```

Comment:
Might be worth saying that this can be Dev or Tech preview, and that the same color is utilized regardless.

Reply:
@serenamarie125 I'll add this in! Thanks for the feedback, here!